interface/super/edit_list.php

```diff
@@ -397,7 +397,7 @@ function writeOptionLine($option_id, $title, $seq, $default, $value, $mapping=''
   } else {
     echo "  <td align='center' class='optcell'>";
     echo "<input type='text' name='opt[$opt_line_no][notes]' value='" .
```

Comment:
with the error message of openemr or for long json of validation its should be input no limited